static/js/vue-cdr-access/src/components/footer.vue

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
I know there isn't much logic in this template, but it would be good to add a simple test to ensure that the footer template loads and maybe verify that the Home link goes to where we expect it to
https://github.com/UNC-Libraries/box-c/pull/1522/files#diff-5b6a675d270ec776826682eab6299b2ba8b1d89a33587b8edfa38ebb09383acc